GA-worksheet.md

```diff
@@ -25,45 +25,51 @@
 
 ##Project #2 Part #2: Style the Header with CSS
 
-1. What advantages does the `<link>` tag have over the `<style>` tag?
+1. What advantages does the `<link>` tag have over the `<style>` tag? The <link> tag lets you include CSS from an external file. 
 
-2. What is normalize.css?
+2. What is normalize.css? It's a CSS file that preserve the CSS through any broswer. 
```

Comment:
It actually resets any default CSS across different browsers, so the CSS you write is preserved through any browser.